### DISCUSSION TITLE
Verifying & Publishing Contracts on Etherscan Rinkeby Testnet

### DISCUSSION THREAD

**Discussion by toddabraham** [smartcontractkit/full-blockchain-solidity-course-py#163] [Q&A] [answered]
What's the process we should follow to verify & publish our contracts to Etherscan Rinkeby testnet?

**Comment by PatrickAlphaC** [comment, marked as answer]
It should be the same! Just do `publish_source=true` when you deploy.

**Reply by toddabraham** [reply]
Thanks @PatrickAlphaC .  Where should `publish_source=true` be included?  Inside the advanced_collectible.sol or in a py script?

**Reply by PatrickAlphaC** [reply]
`contract.deploy({"from":account}, publish_source=true)`

**Comment by jacobbinnie** [comment]
Your **deploy_and_create.py** script should include the **publish_source=True** as shown below. Hope this helps someone else 👍 

<img width="542" alt="Screen Shot 2022-01-19 at 4 36 54 PM" src="https://user-images.githubusercontent.com/83803154/150059421-7026c6b9-eea4-4b4b-bf69-d892da71f702.png">

**Reply by cromewar** [reply]
Nice Tip @jacobbins, thanks
